crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -244,6 +313,16 @@ pub mod error {
     }
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
Just noticed that the file publicly exports an `error` module and this `Error`. That doesn't seem to be intended like this.

Reply:
all errors are now in a same `error` module.